mico-core/src/main/java/io/github/ust/mico/core/model/MicoServicePort.java

```diff
@@ -57,19 +59,23 @@
      * The port number of the externally exposed port.
      */
     @ApiModelProperty(required = true)
-    private int number;
+    @Min(1)
+    @Max(65535)
```

Comment:
make use of a constant

Reply:
In my opinion this is no "magic number" and the usage of a constant does not enhance the readability.
I would prefer to provide an additional validation message:
`@Min(value = 1, message = "Port is only valid between 1 and 65535")`

Reply:
In my opinion, it is somehow a "magic number". That's why I mentioned it here.
This is only my personal feedback - if you change your code or not, depends on you.